gatsby-browser.js

```diff
@@ -20,6 +22,26 @@ const checkDomain = () => {
     )
 }
 
+const updateURLAsPerUserLanguage = () => {
+    const current_path = window.location.pathname
+    const current_hash = window.location.hash
+    const paths = current_path.split('/')
+    const first_path = paths[1]
+    const has_language_in_url = first_path in language_config
```

Comment:
If u use this regex `\/(?=(en|es|fr|id|it|pl|pt|ru|th|vi|tr|zh-cn|zh-tw|bn|ar|ko|ach|de)\/)\1` u can match all supported languages in language config while supporting any type of URLs

Reply:
If you need it to be dynamic u can load the keys only from language_config and serialize it into a string. See screenshot below, it should support all language code
<img width="932" alt="Screenshot 2023-03-16 at 12 11 11 PM" src="https://user-images.githubusercontent.com/75345074/225512331-6b431014-9a95-4c7e-9929-d0045cff7916.png">

Reply:
Hi @yashim-deriv Thanks for mentioning this regex concept over here. I can update my code according to your suggestion but I have a question 👀  . 

What is benefit behind that?
To achieve this thing I need to Serialize that Object then using that I will have dynamic regex and then I will compare it. 
Why not just `in` ? 👀